scripts/train.py

```diff
@@ -0,0 +1,177 @@
+import random
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+import pytorch_lightning as pl
+import torch
+from pytorch_lightning import LightningDataModule
+from sklearn.mixture import GaussianMixture
+from torch.utils.data import DataLoader, Dataset
+
+from opensynth.data_modules.lcl_data_module import LCLDataModule
+from opensynth.models.faraday.gaussian_mixture.prepare_gmm_input import (
+    encode_data_for_gmm,
+)
+from opensynth.models.faraday.new_gmm import gmm_utils
+from opensynth.models.faraday.new_gmm.new_gmm_model import (
+    GaussianMixtureLightningModule,
+    GaussianMixtureModel,
+)
```

Comment:
♻️ Similarly - worth adding a #TODO that this will be deleted later as this is just utility code for testing